src/page/index.js

```diff
@@ -80,6 +81,13 @@ class ReactWorkerDom {
             case REMOVE_EVENT_HANDLERS:
                 node.removeEventHandlers();
                 break;
+            case INVOKE:
```

Comment:
How would responses for `INVOKE` be sent ? I saw that Angular 2 has a similar `INVOKE` concept on the renderer side also, making the channel duplex.